devices.js

```diff
@@ -133,6 +133,11 @@ const configureReporting = {
         const payload = configureReportingPayload('measuredValue', 10, repInterval.HOUR, 5, overrides);
         await endpoint.configureReporting('msPressureMeasurement', payload);
     },
+    pressureExtended: async (endpoint, overrides) => {
+        const scaledValuePayload = configureReportingPayload('scaledValue', 10, repInterval.HOUR, 5, overrides);
+        const scalePayload = configureReportingPayload('scale', 10, repInterval.HOUR, 5, overrides);
```

Comment:
Instead of configuring reporting for it, can we also read it once instead? (also depends on https://github.com/Koenkk/zigbee-herdsman-converters/pull/1614/files#r498788313)